server/controllers/response.ts

```diff
@@ -78,3 +78,22 @@ export function processDeleteResult(req:Request, res: Response, next: NextFuncti
         res.redirect("/account");
     });
 }
+
+//Get Survey Result site
+export function displayResult(req:Request, res: Response, next: NextFunction): void
+{
+    const id = req.params.id;
+
+    ResponseM.find({ question: id }).then(data => {
+        const surveyResponse = data;
+        res.render("index", { title: "Survey Response", page: "surveyresponse", surveyResponses: surveyResponse});
+
+    });
+}
+
+// Get The list of results with a specific id
+export function getSurveyById(responsetitle: string, done: (err: any, res: ResponseM) => void): void
+{
+    // get survey id:db.Survey.find({"_id": SurveyId})
+    ResponseM.find({title: {$eq: responsetitle}},{},{});
```

Comment:
Nice take, but the title is not guaranteed to be unique. You want to use the `_id` field of the survey and do something like:

```ts
ResponseM.find({ question: surveyId }, done);

```

But as written above, there should already be a function that does it.